docker/caffe2/jenkins/common/install_python.sh

```diff
@@ -32,6 +32,7 @@ install_ubuntu() {
       ;;
     3.6)
       install_ubuntu_deadsnakes python3.6-dev
+      apt-get install -y python3-distutils
```

Comment:
Is there `python3.6-distutils`? That would be more accurate.
Also I think the `3.7` branch below needs to install this as well?

Reply:
There is not `python3.6-distutils` available (or `python3.7-distutils`):
```
# apt search python3.6-distutils
Sorting... Done
Full Text Search... Done
# apt search python3.7-distutils
Sorting... Done
Full Text Search... Done
# apt search python3-distutils
Sorting... Done
Full Text Search... Done
python3-distutils/bionic-updates,now 3.6.9-1~18.04 all [installed]
  distutils package for Python 3.x

python3-distutils-extra/bionic 2.41ubuntu1 all
  enhancements to the Python3 build system

```
This seems to be somewhat dependent on the Ubuntu version (i.e., this was not necessary for python3.6 on Ubuntu16.04).

Reply:
>  this was not necessary for python3.6 on Ubuntu16.04
Yeah this is exactly why I was concerning, it can mistakenly install the distutils package from a wrong python version. It's probably not going to be too bad though.

Reply:
My second comment still applies, the `3.7` branch below would also need this install.

Reply:
Yes, I don't think beyond the bloat of a few MBs it should have negative impact since python versions can coexist.

Reply:
Done - I take it respin of the dockers will happen together with update to a more recent ROCm version? If so, note that we'll need #30412 as part of the upgrade.